docs/source/install/deb.rst

```diff
@@ -30,13 +30,12 @@ Install MongoDB, RabbitMQ, and PostgreSQL.
 Setup repositories
 ~~~~~~~~~~~~~~~~~~~
 
-  .. code-block:: bash
+The following script will detect your platform and architecture and setup the repo accordingly. It'll also install the GPG key for repo signing.
```

Comment:
>st2bootstrap-deb.sh - I don't like the idea of wget raw version of the script. I 

Ok, let's punt this until someone complains. 

> Plan to add another notice and migration script when you'll change staging links to production?

Definitely. It will be the same migration script.